src/Microsoft.DotNet.ServiceFabric.ServiceHost/ServiceHost.cs

```diff
@@ -234,11 +234,12 @@ public static void ConfigureDefaultServices(IServiceCollection services)
             services.TryAddSingleton(b => (Microsoft.Extensions.Hosting.IHostingEnvironment) b.GetService<HostEnvironment>());
 #pragma warning restore 618
             ConfigureApplicationInsights(services);
+            services.AddOperationTracking(o => { });
             services.AddLogging(
                 builder =>
                 {
                     builder.AddDebug();
-                    builder.AddFixedApplicationInsights(LogLevel.Information);
+                    //builder.AddFixedApplicationInsights(LogLevel.Information);
```

Comment:
Maybe just remove this? Do we still need to add the regular application insights logger?

Reply:
Line 236 here causes the AI logger to get added.